app/views/superpowers/show.html.erb

```diff
@@ -6,7 +6,7 @@
 
 
 
-<%if !current_user.nil? && current_user.id == @superpower.user_id%>
+<%if !current_user.nil? && current_user.id == @superpower.user_id || !current_user.nil? && current_user.admin? %>
```

Comment:
consider rewriting this conditional:

```ruby
if !current_user.nil? && (current_user = @superpower.user || current_user.admin?) 
```

you could also implement an `editable_by?` instance method on the `Superpower` class. Once implemented, it would simplify the view logic, and you could unit test it that way.